src/wiki/overview/faq.md

```diff
@@ -20,4 +21,12 @@ The philosophy behind the PolyMC project, and wider community as a whole, is to
 
 PolyMC looks to be accessible to everyone on all platforms, so no one is left behind. We do this by officially maintaining packages across a wide range of operating systems such as Windows, macOS, and both native as well as Flatpak and AppImage releases on Linux. And, where there are gaps in official distribution, 3rd party builds will always welcome in our community.
 
-Furthermore, while the MultiMC project is stated to focus on “predictability” without seeking to incorporate any “flashy features”, PolyMC is already shaping up to be quite the opposite. This can already be seen in our recent 1.1.0 release, which brings our new in-launcher mod installer. And this, is just the first of many new changes and features to come.
\ No newline at end of file
+Furthermore, while the MultiMC project is stated to focus on “predictability” without seeking to incorporate any “flashy features”, PolyMC is already shaping up to be quite the opposite. This can already be seen in our recent 1.1.0 release, which brings our new in-launcher mod installer. And this, is just the first of many new changes and features to come. PolyMC 1.4.0 also added the Mod Updater.
+
+### Theming not working
+
+PolyMC switched to Qt 6 in the 1.4.0 update.  
+This means that some themes, like Breeze from KDE will not work.  
+If you have this issue, check if your theme has a Qt 6 version. If it does not, you need to switch to a Qt 5 build.  
+You can find a Qt 5 build on the [download page](https://polymc.org/download/)  
+If you are using Gnome, install the `adwaita-qt6` and `qt6ct` packages (package names might differ between distros), then run `qt6c` from the terminal and select `Adwaita` as the theme. PolyMC should now use it.
```

Comment:
1) why adwaita-qt+qt6ct wtf, just use qgnomeplatform

Reply:
2) polymc didn't switch to qt6 in 1.4.0, but some packages like the ones in the AUR did

Reply:
3) you shouldn't mention gnome at all here, qgnomeplatform should just work

Reply:
Except it didn't for me so

Reply:
https://archlinux.org/packages/community/x86_64/qgnomeplatform-qt6/ 

Do you have this installed?